packages/quantic/force-app/main/default/lwc/quanticSort/e2e/fixture.ts

```diff
@@ -0,0 +1,112 @@
+import {SortObject} from './sortObject';
```

Comment:
```suggestion
import {SortObject} from './pageObject';
```

in line with my other comment